### DISCUSSION TITLE
Network Issue when configuring the application

### DISCUSSION THREAD

**Discussion by georgebastion** [Blair2004/NexoPOS#1507] [Gastro 4.x Ideas & Issues]
I get a network issue on the installation page. When I click complete installation, I get an error "network Error". From this point I cannot continue the installation.

**Comment by Blair2004** [comment]
Hi, if you're using php artisan a serve, please refer to this comment

https://github.com/Blair2004/NexoPOS/discussions/1472#discussioncomment-5815118
